### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by oliviaredfield**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-
-**NOTE**: While there are example calls to the API and demos in this repository, the code for the API is not open source.
+<img align="right" src="img/perspective_icon-2020.png" alt="Perspective API logo" />Perspective is a free API that helps you host better conversations online. The API uses machine learning models to score the perceived impact a comment might have on a conversation. You can use this score to give feedback to commenters, help moderators more easily review comments, allow readers to more easily find interesting or productive comments, and more.
+ 
+Perspective was created by Jigsaw and the Google Counter Abuse Technology team in a collaborative research initiative called [Conversation-AI](https://conversationai.github.io/). Check out PerspectiveAPI.com to learn more about the API. For developer documentation and instructions on how to get started, please visit our Developers site at [support.perspectiveapi.com](https://support.perspectiveapi.com/s/). If you can’t find the answer you’re looking for, [contact us](https://support.perspectiveapi.com/s/contact-us).
```
Comment (sorensenjs) [inline README.md]: Linkify [PerspectiveAPI.com](https://perspectiveapi.com)

File: README.md [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
+Check out [our other repositories](https://github.com/conversationai) for open source experiments, tools, and research data that explore ways to combat online toxicity and harassment. **NOTE**: While there are example calls to the API and demos in this repository, the code for the API is not open source.
```
Comment (sorensenjs) [inline README.md:6]: I might change this to say
"the code for the API and models are a mix of open-source and proprietary components."
We use significant open source code in our service (like tensorflow) so it's a mix.

**Review by sorensenjs — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (asanusi-google) [reply README.md]: +1

File: README.md (hunk 2 above)
Comment (asanusi-google) [reply README.md:6]: Not sure we need this level of granularity and this was the text originally used

**Pull request merged**